polaris-tokens/src/token-groups/typography.ts

```diff
@@ -7,44 +7,29 @@ export const typography = {
     value:
       "ui-monospace, SFMono-Regular, 'SF Mono', Consolas, 'Liberation Mono', Menlo, monospace",
   },
-  'font-size-1': {
+  'font-fontsize-75': {
     value: '12px',
   },
-  'font-size-2': {
-    value: '13px',
-  },
-  'font-size-3': {
+  'font-fontsize-100': {
```

Comment:
**question:** Why did we remove the `-` in `font-size`? This makes it a bit harder for us to preform transforms on the token name in the future.